Makefile

```diff
@@ -40,14 +43,14 @@ test: clean
 	-r requirements/app/test.txt
 
 	# run tests with coverage
-	python -m coverage run --source src/lightning/pytorch -m pytest src/lightning/pytorch tests/tests_pytorch -v
-	python -m coverage run --source src/lightning/app -m pytest tests/tests/app -v
-	python -m coverage run --source src/lightning/fabric -m pytest src/lightning/fabric tests/tests_fabric -v
+	python -m coverage run --source src/pytorch_lightning -m pytest src/pytorch_lightning tests/tests_pytorch -v
+	python -m coverage run --source src/lightning_app -m pytest tests/tests_app -v
+	python -m coverage run --source src/lightning_fabric -m pytest src/lightning_fabric tests/tests_fabric -v
```

Comment:
Is this correct? This is now referencing the standalone shells

Reply:
I think it is but lets rever it for now :)